packages/cli/src/databases/repositories/workflow.repository.ts

```diff
@@ -219,12 +397,11 @@ export class WorkflowRepository extends Repository<WorkflowEntity> {
 		qb: SelectQueryBuilder<WorkflowEntity>,
 		select?: Record<string, boolean>,
 	): void {
-		// Always start with workflow.id
-		qb.select(['workflow.id']);
-
 		if (!select) {
```

Comment:
This changes are to avoid selecting the same field twice. It should not have been part of this PR.